### PR TITLE
[PWD-2561] Page not scrolled to top when navigating via sidebar/navbar

### PR DESCRIPTION
**Pull request by Vesals**
1. adjustment di beberapa page" lain juga biar scroll keatas
2. minor layout adjusment di home page & learning material detail

### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [x] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

File: src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx [hunk 1]
```diff
@@ -46,7 +46,9 @@ const Profile: FC<{ toggleMenu: () => void }> = ({ toggleMenu }) => {
       <Stack p={1} direction="row" spacing={1} sx={{ cursor: 'pointer' }}>
         <ProfileAvatar src={profilePicture} size={40} />
         <Stack minWidth={0}>
-          <Typography variant="subheading3">{`${firstName} ${lastName}`}</Typography>
+          <Typography variant="subheading3">{`${firstName || ''} ${
```
Comment (hiljaker) [inline src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx:49]: ini udah ada default valuenya masih harus pake or kah?

Comment (Vesals) [reply src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx:49]: ini cm biar ilangin warning si zak, apa apus aja???

Comment (hiljaker) [reply src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx:49]: oiya karena first/lastName sendiri punya type string | null yak, keep aja kalau begitu phyn

**Review by hiljaker — APPROVED** [review]

File: src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx (hunk 1 above)
Comment (Vesals) [reply src/components/Navbar/components/MenuProfile/components/MenuProfileItem.tsx:49]: wokai

**Review by hiljaker — APPROVED** [review]

**Review by haikal1223 — APPROVED** [review]

**Pull request merged**